packages/nocms-plugin-json-resources/test/createJsonResourceProvider.spec.js

```diff
@@ -0,0 +1,107 @@
+import assert from 'assert';
+import { createJsonResourceId } from '../lib/createJsonResourceProvider.js';
+import { activate } from '../src/index.js';
+
+const expectedResources = [{
```

Comment:
Should be in a `before` in initial `describe` scope.

Reply:
This needs to be globally available for the fakeFunctions